apicops_mustgather.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# Collecting the output of some some important debugging apicops commands
+
+which apicops &> /dev/null
+if [[ $? -eq 0 ]]; then
+    APICOPS="apicops"
+else
+    if [[ ! -e /tmp/apicops  ]]; then
```

Comment:
Maybe prompt the user before trying to download apicops similar to what is done for `kubectl cnp plugin` https://github.com/ibm-apiconnect/v10-postmortem/blob/master/generate_postmortem.sh#L62